packages/jest-config/src/validConfig.js

```diff
@@ -57,6 +57,9 @@ module.exports = ({
   noStackTrace: false,
   notify: false,
   preset: 'react-native',
+  reporters: [
```

Comment:
nope, `types/Config.js` holds types for Flow (which is stripped with babel for runtime). `jest-validate` checks for actual valid JS object, which in our case is `validConfig.js`.